src/operun/crm/browser/templates/accounts.pt

```diff
@@ -0,0 +1,62 @@
+<html xmlns="http://www.w3.org/1999/xhtml"
+      xml:lang="en"
+      xmlns:tal="http://xml.zope.org/namespaces/tal"
+      xmlns:metal="http://xml.zope.org/namespaces/metal"
+      xmlns:i18n="http://xml.zope.org/namespaces/i18n"
+      lang="en"
+      metal:use-macro="context/main_template/macros/master"
+      i18n:domain="operun.crm">
+  <body>
+
+    <metal:content-core fill-slot="content-core">
+      <metal:content-core define-macro="content-core">
+
+        <div id="accounts-container">
+          <div class="accounts-wrapper" tal:define="accounts python:view.get_accounts()" tal:repeat="account accounts">
```

Comment:
Wrong repeat level, move definition to accounts-container and repeat account-wrapper